.github/workflows/tag_release.yml

```diff
@@ -18,11 +18,39 @@ jobs:
     - name: Set GOVERSION
       run: |
         echo "::set-env name=GOVERSION::$(go version)"
+    - name: Install Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: '2.7'
+    - name: Install github_changelog_generator
+      run: gem install github_changelog_generator
```

Comment:
It might be worth using Bundler with a `Gemfile.lock` to keep the `github_changelog_generator` pinned to a particular version. Alternatively, installing a specific version on this line with `-v` will keep it stable. The only concern here is that the output format may break the consumers of the [*_]CHANGELOG.md files.